packages/measure-tools/README.md

```diff
@@ -0,0 +1,164 @@
+    />
+  );
+```
+
+## What to add in your frontstage
+
+If you are not using the iTwin-Viewer or iTwin App UI, you can add the measurement tools and measurement property widget to your frontstage like so:
+
+### Tools
+
+Similar to  ```CoreTools``` in iTwin.js, measure-tools has a collection of item definitions organized under the ```MeasureToolDefinitions``` singleton. The application can arrange or group the default tool buttons any way it wants.
+Below is an example of creating a vertical toolbar filled with the default measurement tools.
+
+ ```typescript
+const verticalToolbar =
+    <Toolbar
+    expandsTo={Direction.Right}
+    items={
+        <>
+        <ActionItemButton actionItem={MeasureToolDefinitions.measureDistanceToolCommand} />
+        <ActionItemButton actionItem={MeasureToolDefinitions.measureAreaToolCommand} />
+        <ActionItemButton actionItem={MeasureToolDefinitions.measureLocationToolCommand} />
+        <ActionItemButton actionItem={MeasureToolDefinitions.measureRadiusToolCommand} />
+        <ActionItemButton actionItem={MeasureToolDefinitions.measureAngleToolCommand} />
+        </>
+    }
+    />;
+```
+
+### Measurement Property Widget
+
+Most applications will have a property grid widget in the lower right corner of the 9-zone UI. Our measurements do not display properties in the element property grid, instead there is a separate widget that will
+display properties of any selected measurements. It normally is added as a separate tab to the same zone as the property grid. Below is example code from the 9-zone sample application with the measurement widget defined.
+
+In this code, the measurement widget is hidden until a measurement has been selected. With this code you can have measurements and elements selected and have two tabs to switch between properties of either.
+
+ ```typescript
+bottomRight={
+    <Zone defaultState={ZoneState.Open} allowsMerging={true}
+    widgets={[
+
+        <Widget id="Properties" control={PropertyGridWidget} defaultState={WidgetState.Closed} fillZone={true}
+        iconSpec="icon-properties-list" labelKey="MyApp:components.properties"
+        applicationData={{
+            iModelConnection: UiFramework.getIModelConnection(),
+            rulesetId: this._rulesetId,
+        }}
+        syncEventIds={[SyncUiEventId.SelectionSetChanged]}
+        stateFunc={PropertyGridWidget.generateWidgetStateFuncWithMemory()}
+        />,
+
+        <Widget id="Measurements" control={MeasurementWidget} defaultState={WidgetState.Hidden} fillZone={true}
+        iconSpec="icon-measure" labelKey="TestMeasurementApp:components.measurements"
+        applicationData={{
+            iModelConnection: UiFramework.getIModelConnection(),
+        }}
+        syncEventIds={[MeasurementSyncUiEventId.MeasurementSelectionSetChanged]}
+        stateFunc={MeasurementWidget.generateWidgetStateFuncWithMemory()}
+        />,
+    ]}
+    />
+}
+```
+
+![Measurement Widget](./docs/images/MTDocs-Widget.gif)
```

Comment:
Any idea how the gifs will show up in the README on npmjs.org since they're not published?

Reply:
I dont believe they would show up on npmjs, i dont like the idea of publishing the gifs and making the deliverable even larger. Maybe we could just say if gifs don't appear look at the actual readme in github?

Reply:
i updated the gifs to point to github, and according to this [SO post](https://stackoverflow.com/questions/63759426/how-to-show-gif-in-npm-packages-readme-file) should work, we can check once the first ver is released.